code/core/src/types/modules/frameworks.ts

```diff
@@ -1,5 +1,4 @@
 // auto generated file, do not edit
-// the file gets generated by the script ./code/lib/types/scripts/generate-available-frameworks.js
 export type SupportedFrameworks =
```

Comment:
Why was the script, which builds this file, deleted? Do we have a substitution for it in place?